Cluster 2: The Sequel

Squamatic opened the pull request:
<!-- Guidelines: TBD sorry. Sorry. I'm trying to fix it -->

## About the PR
Brings Cluster back into rotation with a new main engine and several small changes to bring it up-to-date with our current mapping requirements.

## Why / Balance
I missed Cluster. She desperately needed a new engine. Lowpop engineers want to play with the supermatter. Surely nothing will go wrong if I combine these ideas.

The mapped pipe layout for the supermatter cuts out a lot of failure points and should allow less experienced engineers more room to experiment, with an emphasis on using gas mixes for activation instead of emitters. I've also included simplified setup instructions based on The Supermatter for Dummies.

This will (hopefully) satisfy demand for lowpop stations with more interesting gameplay for engineering while remaining easy-to-manage in the event that there are no engineers.

## Map Changelog
Added:
- Engineering has been completely reworked to accomodate a supermatter crystal
- Atmos and its associated maints have moved slightly to the right
- Increased the number of SMESes from 3 to 4 and removed the SMES ninja bombing target
- Replaced the old perma brig with genpop
- Emergency radiation protection supplies located in the cargo bay and the maints surrounding engineering
- Expanded solar panels
- Document printers can now be found on the bridge and in the library
- Sentry turrets and their control panels placed in the AI core, armory, and vault
- The kitchen now contains an oven and food-o-mat
- Medical now has access to a medical assembler and medical records computer
- You can now leave medical instead of just closing the shutters
- Salvage has received a job board console
- Minor adjustments to several job spawn locations for more variety

Removed:
- Administrative Assistant
- Brigmedic
- Brig and reworked the front of security (thank you mouse)

## Media
<img width="3232" height="4288" alt="Cluster-0" src="https://github.com/user-attachments/assets/cecad45c-96f4-4474-8de3-5a9919518463" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the Pull Request and Changelog Guidelines. <!-- SORRY WE DONT HAVE THIS ONE YET. SORRY -->
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may g
et your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: Cluster has been updated and re-rotated!

dayn-e commented:
oh cluster my babygirl i missed her so bad

starlighthowls commented:
okay heres some stuff i noticed from testing!

- each department needs their specific cargo order computer and HOP needs the funds computer
- also map some department lockboxes for selling stuff
- you probably don't need that many mice spawners as the vents also will throw them out on events
- might be worth putting more than one copy of your SM guide note around or moving it, it took me a bit to spot it
- runtime is a CMO pet steal target and shouldn't be in engie (use a random cat spawner or poppy!)
- i believe suppy is also a steal target and should be placed
- ive gotten AME part as a steal target before so you may need to put a few in a box somewhere just encase (also it can be used in shuttle making, enrichment!!)
- lots of maints have no lights at all, which is okay just thought id say bcs some parts do have them
- do you have passenger spawners?
- is there a button to open the blast door from the SM to space?
- some of the cameras in engi and atmos are on the general network rather than engi one
- the library needs a book return machine

i missed this map so much im so excited to play again yyEAHHHHHHH

hivehum reviewed (dismissed):
for bookkeeping

Squamatic commented:
MY ECONOMY!!! (Departmental economy stuff isn't listed in either of our mapping resources oops)

The mouse spawners, maints lights, and Runtime spawn are all untouched from the last version of the map that was in rotation. I can move Runtime if he's an issue, but- hang on post canceled how long was this map in rotation with two Morticia spawners...

Will throw a fix together in a couple of hours. I'm so glad people missed Cluster as much as I did!

Squamatic commented:
Added request computers and lockboxes, threw an AME crate in engineering, fixed the pet spawns, placed a few more copies of the SM notes, cameras should all be on the right network now, librarians have enrichment. 👍
